src/java/org/apache/cassandra/config/BitRate.java

```diff
@@ -0,0 +1,331 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.cassandra.config;
+
+import java.util.Arrays;
+import java.util.Objects;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+import com.google.common.primitives.Ints;
+
+/**
+ * Represents a bit rate type used for cassandra configuration. It supports the opportunity for the users to be able to
+ * add units to the confiuration parameter value. (CASSANDRA-15234)
```

Comment:
What if we took these cases where we have CASSANDRA-15234 in parenthesis and use `@since`?

ex. `@since CASSANDRA-15234`

Reply:
I am all in for that but quick grep shows we haven't done it like that before, but doing it the way I did it with () is everywhere. Not sure that I want to change the style now. But I agree I prefer @since

Reply:
Yeah, I mean `@since` is also usually a version number, like 4.1.0. (We could also do that independent of how we mention the Jira.) I'm not wedded to this idea, but it seems like we have an opportunity to "innovate" :)

Reply:
Oh you meant not to remove the ticket in brackets and add the @since version instead but to be like an addition.
@since version and a ticket number. That we can do. I will add it at the end when we really know which version this will land so I don't have to change it more than once